tests/lib/rules/no-useless-constructor.js

```diff
@@ -199,3 +199,169 @@ ruleTester.run("no-useless-constructor", rule, {
 		},
 	],
 });
+
+const ruleTesterTypeScript = new RuleTester({
+	languageOptions: {
+		parser: require("@typescript-eslint/parser"),
+	},
+});
+
+ruleTesterTypeScript.run("no-useless-constructor", rule, {
+	valid: [
```

Comment:
Can we make sure to add all the test cases [from here](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/tests/rules/no-useless-constructor.test.ts)?

Reply:
I meant all TS specific + some basic js because the parser is different. Also, can we add `loc` information assertions on errors?